capstone/src/test/javascript/script.test.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
Do you plan on adding tests here? or in a subsequent PR?

I usually prefer tests to be added in the same PR so that I can see both the source code and the tests.

Reply:
I don't think that I have anything to test yet, but I like having the file there.